packages/auth-providers/dbAuth/middleware/src/__tests__/createDbAuthMiddleware.test.ts

```diff
@@ -522,6 +522,14 @@ describe('createDbAuthMiddleware()', () => {
 
       const serverAuthContext = req.serverAuthContext.get()
       expect(serverAuthContext).toBeNull()
+
+      // mwResponse gets converted to a response eventually.
+      const finalResponse = res.toResponse()
+      expect(finalResponse.headers.getSetCookie()).toEqual([
```

Comment:
To be honest the code comment you had was confusing me more than it was helping me. I suggest just getting rid of it and inlining the `res.toResponse()` part. I think the code speaks for itself here 🙂 

```suggestion
      expect(res.toResponse().headers.getSetCookie()).toEqual([
```